lib/scala/project-manager/src/test/scala/org/enso/projectmanager/infrastructure/languageserver/LanguageServerSupervisorSpec.scala

```diff
@@ -108,9 +104,20 @@ class LanguageServerSupervisorSpec
     fakeServer.stop()
   }
 
-  it should "restart server limited number of times" in new TestCtx {
+  it should "restart server limited number of times" ignore new TestCtx {
```

Comment:
maybe mark `Flaky` instead?

Reply:
The thing is, this test will not work at all currently as we have logic for restarting only once.
I guess I should add a comment clarifying this.

Reply:
Since we decided to abandon restart retries, this test can be removed altogether.